controller/db/migration.go

```diff
@@ -1,20 +1,28 @@
 package db
 
 import (
+	"github.com/netfoundry/ziti-fabric/controller/xt_smartrouting"
+	"github.com/netfoundry/ziti-foundation/storage/ast"
 	"github.com/netfoundry/ziti-foundation/storage/boltz"
 	"github.com/pkg/errors"
+	"strings"
 	"time"
 )
 
-const CurrentDbVersion = 1
+const CurrentDbVersion = 2
 
 func (stores *stores) migrate(step *boltz.MigrationStep) int {
 	if step.CurrentVersion == 0 {
 		stores.migrateToV1(step)
 		return 1
 	}
 	if step.CurrentVersion == 1 {
-		return 1
+		stores.extractTerminators(step)
+		return 2
+	}
+
+	if step.CurrentVersion == 2 {
+		return 2
 	}
 	step.SetError(errors.Errorf("Unsupported fabric datastore version: %v", step.CurrentVersion))
```

Comment:
Super minor nit... but I'm pretty sure Goland's linter complains about error messages that start with a capital letter.